docs/topics/testing/advanced.txt

```diff
@@ -781,6 +781,72 @@ utility methods in the ``django.test.utils`` module.
     configuration that need to be reversed. It's the return value of the
     :meth:`setup_databases` method.
 
+.. function:: isolate_apps(*app_labels, attr_name=None, kwarg_name=None)
```

Comment:
I don't think it's the right place to document `@solate_apps`. These functions are described as:
```
To assist in the creation of your own test runner, Django provides a number of
utility methods in the ``django.test.utils`` module.
```
I would move it to the new paragraph e.g. "Isolating apps" in the ["Test cases features"](https://docs.djangoproject.com/en/dev/topics/testing/tools/#test-cases-features).